recipes/typechecks/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: typechecks
+  version: "0.1.0"
+
+source:
+  fn: typechecks-0.1.0.tar.gz
+  url: https://files.pythonhosted.org/packages/62/21/15129201c1f52f6af1e7809e96dce46da4b406c2c07fe9425e92f51edc5c/typechecks-0.1.0.tar.gz
+  md5: 140956d8f5991b9b73769431b95aeff8
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Is there a reason to skip windows? There doesn't appear to be any reference to it not being supported that I can find.